cli/wallet/wallet.go

```diff
@@ -397,6 +415,55 @@ func importWallet(ctx *cli.Context) error {
 	return nil
 }
 
+func removeAccount(ctx *cli.Context) error {
+	wall, err := openWallet(ctx.String("path"))
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+	defer wall.Close()
+
+	addrFlag := ctx.Generic("addr").(*flags.Address)
+	if !addrFlag.IsSet {
+		return cli.NewExitError("address must be provided", 1)
+	}
+	addr := addrFlag.Uint160()
+	acc := wall.GetAccount(addr)
+	if acc == nil {
+		return cli.NewExitError("account wasn't found", 1)
+	}
+
+	if !ctx.Bool("force") {
+		fmt.Printf("Account %s will be removed. This action is irreversible.\n", addrFlag.String())
+		if ok := askForConsent(); !ok {
+			return nil
+		}
+	}
+
+	if err := wall.RemoveAccount(acc.Address); err != nil {
+		return cli.NewExitError(fmt.Errorf("error on remove: %v", err), 1)
+	} else if err := wall.Save(); err != nil {
+		return cli.NewExitError(fmt.Errorf("error while saving wallet: %v", err), 1)
+	}
+	return nil
+}
+
+func askForConsent() bool {
+	// Probably there are better ways to ask.
+	// See https://www.dartmouth.edu/consent/communication/howdo.html
```

Comment:
I think this comment may confuse people a bit, we ought to come up with one and only proper way to ask for consent here, commit to it and stick with it.